File: src/Domain/Entities/Playlist.cs
namespace Domain.Entities;

public class Playlist
{
    public int Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public int CreatorId { get; set; }
    public DateTime CreatedAt { get; set; }

    public User? Creator { get; set; }
    public ICollection<PlaylistItem> Items { get; set; } = new List<PlaylistItem>();
}

File: clients/AndroidViewerApp/app/src/main/java/com/example/streaming/models/Playlist.java
package com.example.streaming.models;

import java.util.List;

public class Playlist {
    private int id;
    private String title;
    private String description;
    private int creatorId;
    private String createdAt;
    private List<PlaylistItem> items;

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }

    public List<PlaylistItem> getItems() {
        return items;
    }
}
